requirements.txt

```diff
@@ -1,3 +1,5 @@
 requests>=2.20.0
 pyodata==1.7.0
 PyYAML==5.4.1
+
+urllib3~=1.26.7
```

Comment:
Isn't urllib3 needed only if a user wants to use HTTP over SSL?

Reply:
True. I removed it must have slipped in with the rebasing.